lib/streamlit/web/server/browser_websocket_handler.py

```diff
@@ -61,6 +61,25 @@ def write_forward_msg(self, msg: ForwardMsg) -> None:
         except tornado.websocket.WebSocketClosedError as e:
             raise SessionClientDisconnectedError from e
 
+    def select_subprotocol(self, subprotocols: List[str]) -> Optional[str]:
```

Comment:
To clarify: This method is expected to always return `streamlit` (since this is the first specified protocol) or None if no subprotocols are specified? The actual method that extracts the token is not implemented yet in this PR, or?

Reply:
Yep, that's correct. The method should either return `streamlit` (since it's always the first specified protocol) or `None`.

The actual method that extracts the token isn't implemented in this PR, and will be implemented in the `feature/session-manager` branch, but we want this change to land before we use the auth token ourselves since it's needed in the SiS world.